Test/exceptions/TestAttributeErrors.dfy

```diff
@@ -0,0 +1,12 @@
+// RUN: %dafny "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+include "../exceptions/VoidOutcomeDt.dfy"
+
+function {:test} PassingTest(): VoidOutcome {
```

Comment:
I suggest adding a comment at the end of this line to indicate that you expect an error. (Same on L10.) I find this makes it easier to read just the .dfy test file.